Dockerfile

```diff
@@ -47,7 +51,23 @@ RUN \
   cd swoole-src && \
   git checkout $PHP_SWOOLE_VERSION && \
   phpize && \
-  ./configure --enable-sockets --enable-http2 && \
+  ./configure --enable-http2 && \
+  make && make install && \
+  cd .. && \
+  ## Imagick Extension
+  git clone https://github.com/Imagick/imagick && \
+  cd imagick && \
+  git checkout $PHP_IMAGICK_VERSION && \
+  phpize && \
+  ./configure && \
+  make && make install && \
+  cd .. && \
+  ## YAML Extension
+  git clone https://github.com/php/pecl-file_formats-yaml && \
```

Comment:
The official YAML Docker container uses `https://pecl.php.net/get/yaml-2.2.0b2.tgz` instead of the Git version.

Reply:
Pecl was deprecated from the official Docker image: https://github.com/docker-library/php/issues/1029

Reply:
That's something else ;)

Reply:
Yeh I know, just trying to keep the codebase consistent, I'll improve docs later.